src/main/java/mate/academy/spring/dto/request/CinemaHallRequestDto.java

```diff
@@ -1,7 +1,12 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.Min;
+import javax.validation.constraints.Size;
+
 public class CinemaHallRequestDto {
+    @Min(10)
     private int capacity;
+    @Size(min = 5, max = 50)
```

Comment:
```suggestion
    @Size(min = 5, max = 255)
```
50 is not enough for description